clients/src/main/java/org/apache/kafka/common/requests/AlterConfigsRequest.java

```diff
@@ -87,6 +87,16 @@ public Builder(Map<ConfigResource, Config> configs, boolean validateOnly) {
         public AlterConfigsRequest build(short version) {
             return new AlterConfigsRequest(data, version);
         }
+
```

Comment:
In general we don't define equals or hashCode on these builders.  Why are we defining it here?

Reply:
The purpose is for the mock tests to compare the expected builder in `KafkaApisTest`